reloading/reloading.go

```diff
@@ -43,37 +109,66 @@ func (withId *wrapWithIdPage) Containers(r *http.Request) (cs []ct.Container, er
 	if err != nil {
 		return
 	}
+
+	var cbt *containersByTag
+	if val := r.Context().Value(containersByTagKey); val != nil {
+		cbt = val.(*containersByTag)
+	}
+
 	for i, oc := range ics {
-		cw := cb.Wrap("div", cb.Attrs{"data-container-id": fmt.Sprintf("%d", i)}, oc)
+		cw := withId.wrapWithId(oc, fmt.Sprintf("%d", i+1), cbt)
 		cs = append(cs, cw)
 	}
-	cs = append(cs, cb.Wrap("script",
-		cb.Attrs{"src": "https://cdnjs.cloudflare.com/ajax/libs/fetch/1.0.0/fetch.min.js"}))
-	cs = append(cs, cb.ScriptByString(reloadscript))
+
 	return
 }
 
-func ReloadablePageHandler(page ct.Page, layout ct.Layout) http.Handler {
-	return &reloadableHandler{handler: ct.PageHandler(&wrapWithIdPage{page}, layout), page: page}
-}
+func (withId *wrapWithIdPage) wrapWithId(pc ct.Container, containerId string, cbt *containersByTag) (rc ct.Container) {
```

Comment:
This method will travel through the whole containers tree which setup in a page, and collect those reloadable containers into a store `containersByTag`, And because each container if self renderable without depending parent container. So when a request want to load just the inner container. It can give it's html back.